h2o-py/h2o/h2o.py

```diff
@@ -591,6 +593,7 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     :param na_strings: A list of strings, or a list of lists of strings (one list per column), or a dictionary
         of column names to strings which are to be interpreted as missing values.
     :param skipped_columns: an integer lists of column indices to skip and not parsed into the final frame from the import file.
+    :param custom_non_data_line_markers: If a line in imported file starts with any character in given string it will NOT be imported. Empty string means all lines are imported, None means that default behaviour for given format will be used
```

Comment:
```suggestion
    :param custom_non_data_line_markers: If a line in an imported file starts with any character in the given string, it will be skipped and NOT be imported. An empty string means all lines are imported, None means that default behavior for given format will be used.
```